### DISCUSSION TITLE
[Projects Beta] [Enhancement] Allow to add pictures to draft

### DISCUSSION THREAD

**Discussion by danielinux7** [community/community#9908] [Projects and Issues]
It would be good to be able to add pictures to drafts, I need to include diagrams as a quick reference.

**Comment by danielinux7** [comment]
It seems the functionality is already there but there is no attach files by dragging and dropping.

**Reply by jdubhub** [reply]
Can you advise how to add images? thanks

**Reply by danielinux7** [reply]
Hello,
It's a little hacky, but the way I did it is to add an image to any issue, then copy the generated link and use it in the draft.
Here is an example of a draft (Аҭел Адиаграмма): https://github.com/users/danielinux7/projects/2/views/5

The link looks something like this: 
`![anana-Page-2 (2)](https://user-images.githubusercontent.com/2645266/148753396-cf0325c6-e642-4d24-9034-852dc22c7329.jpg)`
![anana-Page-2 (2)](https://user-images.githubusercontent.com/2645266/148753396-cf0325c6-e642-4d24-9034-852dc22c7329.jpg)

**Reply by jdubhub** [reply]
Thanks for sharing!

**Reply by ANILTEKAM** [reply]
SREE VANKTESH AQUA WATERPROOFING.COM

**Comment by rmcsharry** [comment]
> 

![image](https://user-images.githubusercontent.com/1156100/162616316-3c5e9cad-16f1-4ceb-98a5-ef1a84321363.png)
